Snabble/UI/Onboarding/Model/OnboardingModel.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  OnboardingModel.swift
+//  Onboarding
+//
+//  Created by Uwe Tilemann on 05.08.22.
+//
+
+import Foundation
+
+#if canImport(Combine)
+import Combine
+
+public final class OnboardingModel: ObservableObject, Decodable {
+    public static let shared : OnboardingModel = load("onboardingData.json")
```

Comment:
Benötigen wir innerhalb des SDK ein Sample für das Onboarding Model? Ich würde diese Logik komplett aus dem Interface des SDK herausziehen und in die Example App nehmen. Warum siehst du es notwendig im SDK?

Reply:
Warum kein minimales Onboarding im SDK? Ich dachte eine Example App sollte so wenig wie möglich Konfiguration enthalten.

Reply:
Das was z.B. die Snabble App als Onboarding benutzt, könnte so auch für die Example App verwendet werden.